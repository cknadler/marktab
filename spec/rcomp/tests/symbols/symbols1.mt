# transitions to a symbol
def: (1:2 2:2)

1:1 / def 
