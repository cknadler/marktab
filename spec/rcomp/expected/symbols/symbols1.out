e|-1-/-2-|
B|---/-2-|
G|-------|
D|-------|
A|-------|
E|-------|
